### PR TITLE
Add a "Give Feedback" option in Help menu

### PR DESCRIPTION
**Pull request by RJ722**
Summary of the changes in this pull request:

As mentioned in TODOs, I've added a "Give Feedback" option in Help menu.

(Copy of #550)

Copied suggestion:

I think that it would be better to redirect the user to `.../discussions/new/?category=ideas`. @jendrikseipp what do you think?

### CODE REVIEW

File: rednotebook/gui/menu.py [hunk 1]
```diff
@@ -257,6 +258,16 @@ def get_actiongroup(self):
                     _("Fill out a short form about the problem"),
                     self.on_report_bug,
                 ),
+                (
+                    "GiveFeedback",
+                    None,
+                    _("Give Feedback"),
+                    None,
+                    _(
+                        "Fill out a short form and help us know how we can improve RedNotebook for you"
```
Comment (jendrikseipp) [inline rednotebook/gui/menu.py:267]: Let's shorten this to "How can we improve RedNotebook?"

**Review by jendrikseipp — CHANGES_REQUESTED** [review]
Code looks great, thanks! Can you also add a changelog entry, please?

**Comment by jendrikseipp** [comment]
I think the URL in the R is fine. It's good to show existing discussions before starting a new one.

**Comment by RJ722** [comment]
Thanks for the quick review! Added the changelog entry, and changed the sentence.

File: rednotebook/gui/menu.py [hunk 2]
```diff
@@ -257,6 +258,14 @@ def get_actiongroup(self):
                     _("Fill out a short form about the problem"),
                     self.on_report_bug,
                 ),
+                (
+                    "GiveFeedback",
+                    None,
+                    _("Give Feedback"),
+                    None,
+                    _("How we can improve RedNotebook?"),
```
Comment (jendrikseipp) [inline rednotebook/gui/menu.py:266]: we can -> can we

Comment (RJ722) [reply rednotebook/gui/menu.py]: Oops, dome!

**Comment by jendrikseipp** [comment]
Thanks, Rahul!
